modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -104,6 +106,28 @@
 		addBinding: function(){
 			var _this = this;
 
+			this.bind('KalturaSupport_ThumbCuePointsReady' , function(){
+				if (_this.getPlayer().isLive()){
+					//Forcing 1st slide is only on VOD entry
+					return;
+				}
+				// on non-autoplayed player - check if required to force-show the 1st CP
+				var chaptersRawData = _this.getCuePoints();
+				//Since imagePlayer doesn't have a uiconf element - copy the value from chapters plugin which has representation in the uiconf;
```

Comment:
@eitanavgil when you init a class that extends from KBasePlugin, you pass to it a callback function as the second argument: https://github.com/kaltura/mwEmbed/blob/master/modules/KalturaSupport/resources/mw.KBasePlugin.js#L6
and then you can set it's default settings there, see example here(and also all over dualScreen plugin(that initializes imagePlayer component as well):
https://github.com/kaltura/mwEmbed/blob/2b0253b80c4ca0464c89cc9a5c3dc8ee61ca3265/modules/KalturaSupport/components/dualScreen/dualScreen.js#L488

So you can just add the desired config option under the dualScreen config, and when dualScreen initializes it's child component - imagePlayer - just pass the config there:
https://github.com/kaltura/mwEmbed/blob/2b0253b80c4ca0464c89cc9a5c3dc8ee61ca3265/modules/KalturaSupport/components/dualScreen/dualScreen.js#L876...L883
```javascript
var passthroughConfig = {
  prefetch: this.getConfig('prefetch'),
  cuePointType: this.getConfig('cuePointType'),
  showFirstSlideOnLoad: this.getConfig('showFirstSlideOnLoad')
};
```